client/client.go

```diff
@@ -34,25 +38,40 @@ func main() {
 	udp6 := flag.Bool("udp6", false, "Whether to use IPv6")
 	flag.Parse()
 
+	var flagValues = flags{
+		address:  *address,
+		password: *password,
+		udp6:     *udp6,
+	}
+
+	// Testing with Old Config
+	testPorts(flagValues, 5688, 5689)
+
+	// Testing with New Config
+	testPorts(flagValues, 5683, 5684)
+
+}
+
+func testPorts(flagValues flags, udpPort int, dTLSPort int) {
+
 	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(10)*time.Second)
 	defer cancel()
 
-	udpPort := 5688
-	dTLSPort := 5689
 	var udpAddr string
 	var dtlsAddr string
+	_ = dtlsAddr
```

Comment:
Why is this needed?

Reply:
yeah, it crept in while testing locally i guess. Removed it. :-)